src/AutoMapper/Internal/ExpressionFactory.cs

```diff
@@ -21,6 +21,8 @@ public static Expression GetSetter(MemberExpression memberExpression)
 
         public static MethodInfo Method<T>(Expression<Func<T>> expression) => ((MethodCallExpression) expression.Body).Method;
 
+        public static MethodInfo Method<TType, TResult>(Expression<Func<TType, TResult>> expression) => ((MethodCallExpression)expression.Body).Method;
+
```

Comment:
There is some duplication here. Both methods could forward to another one that does the job.